src/components/Pagination.js

```diff
@@ -0,0 +1,41 @@
+const Pagination = (props) => {
+  const arr = ['1', '2', '3', '4', '5'];
+  const filters = document.querySelectorAll('.sidebar__link');
+  const pags = document.querySelectorAll('.pag__link');
+  const filterUrl = 'v2.1/films/search-by-filters?genre=';
+  return (
+    <div class="pagination">
+      {arr.map((i) => {
+        return (
+          <a
```

Comment:
Always move elements to standalone components when you use it in jsx & inside `map`, `filter` etc